Mlem/Views/Tabs/Inbox/Inbox View.swift

```diff
@@ -146,26 +195,17 @@ struct InboxView: View {
                         MessagesFeedView(messageTracker: messageTracker)
                     }
                 }
-            }
-            .fancyTabScrollCompatible()
-            .refreshable {
-                // wrapping in task so view redraws don't cancel
-                // awaiting the value makes the refreshable indicator properly wait for the call to finish
-                await Task {
-                    await refresh()
-                }.value
-            }
-            .hoistNavigation {
-                withAnimation {
-                    scrollProxy.scrollTo(scrollToTop)
+            } header: {
+                VStack(spacing: 0) {
+                    BubblePicker(InboxTab.allCases, selected: $curTab) { tab in
+                        Text(tab.label)
+                    }
+                    .background(Color.systemBackground.opacity(scrollToTopAppeared ? 1 : 0))
+                    .background(.thickMaterial)
```

Comment:
```suggestion
                    .background(.bar)
```

Maybe this is nicer?

Reply:
Yep, that's definitely better